app/Console/ReplayCommand.php

```diff
@@ -28,6 +28,7 @@ class ReplayCommand extends AbstractCommand
     public const OPTION_START_ID = 'start-id';
     public const OPTION_DELAY = 'delay';
     public const OPTION_CDBID = 'cdbid';
+    public const OPTION_DISABLE_AMQP_PUBLICATION_POST_REPLAY = 'disable-amqp-publication-post-replay';
```

Comment:
Maybe a name that fits the intercepting mechanism is `disable-amqp-publication-of-intercepted-messages`?

Reply:
The intercepting felt like an implementation detail to me. When discussing the functionality with Kristof we always talked about it in terms of "dispatching 1 message per replayed or related event/place/organizer, after the replay". I didn't even think of the "intercepting" as an easy implementation for that at first, I even started an implementation that stored the replayed ids and then queried the related repositories etc 😄 So to summarize, I'm not sure the "intercepting" part would be clear for someone who uses the command without knowing the code details.

In hindsight the "post-replay" could maybe be changed to "after-replay" for clarity though.

Reply:
I have renamed it to `--no-amqp-publication-after-replay`, that is also more consistent with the standard `--no-ansi` and `--no-interaction` options that Symfony Console provide